re.sonny.Workbench.Devel.json

```diff
@@ -71,6 +77,32 @@
         }
       ]
     },
+    {
+      "name": "cargo-c",
+      "buildsystem": "simple",
+      "build-commands": ["cargo install cargo-c --root /app"],
+      "build-options": {
+        "build-args": ["--share=network"]
+      },
+      "cleanup": ["*"]
+    },
+    {
+      "name": "gst-plugins-rs",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs",
+          "branch": "main"
+        }
+      ],
+      "build-options": {
+        "build-args": ["--share=network"]
+      },
+      "build-commands": [
+        "cargo cinstall --features wayland,x11glx,x11egl -p gst-plugin-gtk4 --prefix=/app"
```

Comment:
do we need both x11glx and x11egl?

Reply:
the demo works on x11 without glx so we can ship only egl since that is newer.

Reply:
When you say "works on x11" you mean for yourself?

I found this https://mozillagfx.wordpress.com/2021/10/30/switching-the-linux-graphics-stack-from-glx-to-egl/
It looks like we should keep x11glx enabled as it improves compatibility, I will add it back